app.py

```diff
@@ -40,6 +44,16 @@ def request_about():
     data = {"all_users": authentication.get_all_logged_in_users(), "len": len(authentication.get_all_logged_in_users())}
     return render_template("div_templates/about.html", **data)
 
+@app.route("/about_ingredients", methods=['GET'])
+def get_about_ingredients():
+    path = "./static/ingredients/"
+    ingredient_files = []
+    for file in listdir(path):
+        if isfile(path + file):
+            ingredient_files.append(path + file)
```

Comment:
Instead of concatenating file names, it's safer to use [`os.path.join`](https://docs.python.org/3/library/os.path.html#os.path.join).